code/game/gamemodes/cult/ritual.dm

```diff
@@ -239,24 +247,37 @@ This file contains the arcane tome files as well as innate cultist emergency com
 			break
 	if(!rune_to_scribe)
 		return
+	var/turf/Thenewturfyouwalkedto = get_turf(user) //we may have moved. adjust as needed...
+	if(locate(/obj/effect/rune) in Thenewturfyouwalkedto)
+		user << "<span class='cult'>There is already a rune here.</span>"
+		return 
 	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated())
 		return
 	user.visible_message("<span class='warning'>[user] cuts open their arm and begins writing in their own blood!</span>", \
 						 "<span class='cult'>You slice open your arm and begin drawing a sigil of the Geometer.</span>")
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
 		C.apply_damage(0.1, BRUTE, pick("l_arm", "r_arm"))
-		if("Call Forth The Geometer" == entered_rune_name)
+		if("Summon Nar-Sie" == entered_rune_name)
 			C.apply_damage(40, BRUTE, pick("l_arm", "r_arm"))
 			var/area/A = get_area(src)
 			var/locname = initial(A.name)
-			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual before it reaches a critical point.","Central Command Higher Dimensionsal Affairs")
-			if(!do_after(user, 500, target = get_turf(user)))
+			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual at all costs!","Central Command Higher Dimensionsal Affairs", 'sound/AI/spanomalies.ogg')
+			for(var/turf/B in orange (1, user))
+				var/obj/machinery/shield/N = new(B)
+				N.name = "Rune-Scriber's Shield"
+				N.desc = "A potent shield summoned by cultists to protect them while they prepare the final ritual"
+				N.color = "red"
+				N.health = 60
+			if(!do_after(user, 400, target = get_turf(user)))
+				for(var/obj/machinery/shield/U in orange (1, user))
+					qdel(U)
 				return
 	if(!do_after(user, 50, target = get_turf(user)))
```

Comment:
if you fail this do_after the shields won't disappear, so you need the shield removal code here, too.